frontend/src/bundles/common/components/input/input.tsx

```diff
@@ -21,19 +22,27 @@ const Input = <T extends FieldValues>({
     errors,
     label,
     name,
-    placeholder = '',
     type = 'text',
+    isDisabled = false,
+    placeholder = '',
 }: Properties<T>): JSX.Element => {
     const { field } = useFormController({ name, control });
 
     const error = errors[name]?.message;
     const hasError = Boolean(error);
 
     return (
-        <label>
-            <span>{label}</span>
-            <input {...field} type={type} placeholder={placeholder} />
-            {hasError && <span>{error as string}</span>}
+        <label className="flex h-20 flex-col text-sm">
```

Comment:
```suggestion
        <div className="flex h-20 flex-col text-sm">
```
Since you put input inside label whole area of label is clickable for select input, and the problem is that label takes all width of parent element. 
I think only click on input should select input, so you should use div instead of label
![2024-02-13_11-29](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/26843161/36e58bf4-148d-481a-8617-3e4454b088c2)

Reply:
@stpalamar This is what the label exists for, so that a click on the label will focus on the input